docs/serving/istio-authorization.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Enabling requests to Knative services when additional authorization policies are enabled"
+weight: 25
+type: "docs"
+---
+
+Knative Serving system pods, such as the activator and autoscaler components, require access to your deployed Knative services.
+If you have configured additional security features, such as Istio's authorization policy, you must enable access to your Knative service for these system pods.
+
+## Before you begin
+
+You must meet the following prerequisites to use Istio AuthorizationPolicy:
+
+- [Enabling istio sidecar injection](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/).
+- Using net-istio for your Knative Ingress.
+
+## Enabling Istio AuthorizationPolicy
+
+For example, the following authorization policy denies all requests to workloads in namespace serving-tests.
+
+```
+$ cat <<EOF | kubectl apply -f -
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: deny-all
+  namespace: serving-tests
+spec:
+  {}
+EOF
+```
+
+In addition to allowing your application path, you must whitelist the system pods in your Istio AuthorizationPolicy
+to enable access to your application for requests from system pods.
+You can enable access by:
+[Allowing access from system pods by paths](#allow-access-from-system-pods-by-paths).
+[Allowing access from system pods by namespace](#allow-access-from-system-pods-by-namespace).
```

Comment:
This could read slightly easier. Maybe something like:
```suggestion
You can allow access from system pods
[by paths](#allow-access-from-system-pods-by-paths) or 
[by namespace](#allow-access-from-system-pods-by-namespace).
```